pyveg/src/download_modules.py

```diff
@@ -268,15 +273,25 @@ def prep_images(self, dataset):
         dataset = apply_mask_cloud(dataset, self.collection_name, self.cloudy_pix_flag)
         # Take median
         image = dataset.median()
+        bands_to_select = self.RGB_bands
 
         if self.ndvi:
             # Calculate NDVI
             image = add_NDVI(image, self.RGB_bands[0], self.NIR_band)
             # select only RGB + NDVI bands to download
-            bands_to_select = self.RGB_bands + ["NDVI"]
-        else:
-            bands_to_select = self.RGB_bands
+            bands_to_select = bands_to_select + ["NDVI"]
+
+        if self.count:
+            # create pixel level count image
+            count = dataset.count()
+            # sum counts on each band into one image
+            image_count = count.select(self.RGB_bands[0]).rename("COUNT")
```

Comment:
Just to check - this is the image count, which is now based on the red band only? I think that this is a pragmatic solution.

Reply:
Yes, decided to pick the first band, but B2, B3 and B4 would give identical results.